osadchyi_homework_1.cpp

```diff
@@ -0,0 +1,65 @@
+#include <cmath>
+#include <iostream>
+
+int main() {
+  int atmpts{3};
+  constexpr short lwr_bndr = 2;
+  constexpr short uppr_bndr = 255;
+
+  std::cout << "Enter three numbers in range " + std::to_string(lwr_bndr) +
+                   "..." + std::to_string(uppr_bndr) +
+                   " separated by space in order a b c.\nIf value is empty "
+                   "leave 0."
+            << std::endl;
+
+  while (atmpts) {
+    short a{};
+    short b{};
+    short c{};
+    bool a_range{false};
+    bool b_range{false};
+    bool c_range{false};
+
+    std::cin >> a >> b >> c;
+    a_range = a >= lwr_bndr && a <= uppr_bndr;
```

Comment:
старайтесь совмещать объявление и инициализацию переменной, т.е. зачем выше объявлять переменную а тут ее инициализировать
```suggestion
   bool a_range = a >= lwr_bndr && a <= uppr_bndr;
```